XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1020,6 +1020,29 @@
             "Image"
           ]
         },
+        {
+          "UniqueId": "InfoBar",
+          "Title": "InfoBar",
+          "Subtitle": "An inline message to display app-wide status change information",
+          "ImagePath": "ms-appx:///Assets/DefaultIcon.png",
```

Comment:
Maybe we could use the TeachingTip icon until we have a dedicated icon for this? @mdtauk in case you are interested.

Reply:
I did make a set of icons to be used in Visual Studio and the Xaml gallery, but I don't think they were approved.

Reply:
I'd like to avoid confusion between TeachingTip and InfoBar since they are similar sorts of controls. Other ideas: MenuBar, CommandBarFlyout.

Reply:
@mdtauk Could you link to that set of icons? I don't think I've seen the thread on those yet, thanks.

Reply:
> @mdtauk Could you link to that set of icons? I don't think I've seen the thread on those yet, thanks.

https://github.com/microsoft/Xaml-Controls-Gallery/issues/504

Reply:
I switched it to MenuBar for the time being, I'll take a todo on creating something custom 😊